tests/TestCase/Utility/InflectorTest.php

```diff
@@ -204,80 +218,94 @@ public function testSingularizeMultiWordIrregular()
     /**
      * testInflectingPlurals method
      *
+     * @dataProvider pluralizeProvider
      * @return void
      */
-    public function testInflectingPlurals()
+    public function testInflectingPlurals($plural, $singular)
+    {
+        $this->assertEquals($plural, Inflector::pluralize($singular));
```

Comment:
Should we use the 3rd arg to insert some meaningful error message like `$singular . ' inflection failed'` etc?
Otherwise for dataProvider input it could be hard to know which row exactly failed maybe.